dali/operators/decoder/image_decoder.cc

```diff
@@ -54,6 +54,19 @@ will occur during the pipeline execution. One way to find the ideal value is to
 run over the dataset with the ``memory_stats`` argument set to True and then copy the largest
 allocation value that was printed in the statistics.)code",
       16*1024*1024)
+  .AddOptionalArg("device_memory_padding_jpeg2k",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+The padding for nvJPEG2k's device memory allocations, in bytes. This parameter helps to avoid
+reallocation in nvJPEG2k when a larger image is encountered, and the internal buffer needs to be
+reallocated to decode the image.
+
+If a value greater than 0 is provided, the operator preallocates one device buffer of the
```

Comment:
"one device buffer of the requested size per thread" -> Not true. See allocations below.

Reply:
Done